files/settings-default.cfg

```diff
@@ -145,6 +145,10 @@ difficulty = 0
 # Show duration of magic effect and lights in the spells window.
 show effect duration = false
 
+# Do not allow to add same effect multiple times in spell creation
+# Example: [fire damage 10pts for 1s]+[fire damage 1-2pts for 10s] is possible if disabled
+unique spell effects = true
```

Comment:
Indeed. If one still has 0.35 settings-default.cfg, that means they have installed 0.41 improperly.

Also, `Settings::Manager::getBool("unique spell effects", "Game")` will throw, not return false, when the option is missing:
```
    throw std::runtime_error(std::string("Trying to retrieve a non-existing setting: ") + setting
                             + ".\nMake sure the settings-default.cfg file was properly installed.");
```